aikiblog/templates/add_techniques.html

```diff
@@ -50,7 +50,7 @@
 			<td>{{ technique.date|date:" d.m.Y " }}&nbsp;&nbsp;</td>
 			<td>{{ technique.stand }}&nbsp;&nbsp;</td>
 			<td>{{ technique.attack }}&nbsp;&nbsp;</td>
-			<td>{{ technique.technique }}&nbsp;&nbsp;</td>
+			<td><a href="{% url "technique-list" technique.technique.slug %}">{{ technique.technique }}</a>&nbsp;&nbsp;</td>
```

Comment:
a jeśli chodzi o te hasze to nie czaje tego nazewnictwa. Jak mnie uczyłaś, to mówiłaś, że każde zadanie musi mieć swojego pull requesta. Czyli nie nazywać ich swoimi numermi, tylko odnosić się do tych, które nadaje github? I gałęzie nazywać też numerkami, które będą na danym pull requeście?